dbt_copilot_helper/utils/validation.py

```diff
@@ -376,6 +376,7 @@ def is_between(value):
             ENV_NAME: {
                 Optional("plan"): RDS_PLANS,
                 Optional("volume_size"): int_between(20, 10000),
+                Optional("iops"): int_between(1000, 2000),
```

Comment:
Unfortunately I think the validation needs to be a bit more complex than this, there are some extra constraints listed under the [iops property here.](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html)

Constraints:
- For RDS for Db2, MariaDB, MySQL, Oracle, and PostgreSQL - Must be a multiple between .5 and 50 of the storage amount for the DB instance.

Reply:
I'm just picking this up in Yusuf's absence. As @antroy-madetech suggests 9950 is still well below the upper limit boundary so am I OK to amend to this or also as suggested just remove the upper limit?